### DISCUSSION TITLE
GitHub Co-Pilot loading error

### DISCUSSION THREAD

**Discussion by sleepingcat4** [community/community#35370] [Copilot Conversations] [answered]
Lately, I'm encountering some problem with my GitHub Co-Pilot. Without any valid reason it fails to load on my VS Code. And wants me to update to for using it. 

But, when I tried to update and checked the logs it showed me that, GitHub Co-Pilot extension for VS Code was in the latest version that was released 3 months ago. Roughly version 1.31. 

Can someone knows any hack to tackle with this bug?

**Comment by sleepingcat4** [comment]
**Update** I'm currently using 1.30V of GitHub Copilot and it's working without any problems. Still, someone explain if I'm only one whose experiencing? and how to tackle the bug!

**Comment by davecheney** [comment, marked as answer]
@sleepingcat4 please update to VSCode 1.71 or later, then newer versions of Copilot will be available.
